### DISCUSSION TITLE
Somes features suggestions to develop

### DISCUSSION THREAD

**Discussion by medhidev** [stoatchat/discussions#638] [Feature Suggestions]
Hi everyone ^^ ! here are some varied proposals, some of which should have already been mentioned, but I wanted to make a summary of all the features which seem interesting to me to develop.

## PROFILE
- Activity profile
- better access to status change → the circle on the profile photo & and add a status comment
- Screencast → Screen sharing (we can use streaming technology like Twitch with a lower video resolution so as not to overload the servers)
- Camera in vocals
- Integration of API links on user profiles (example = display the weather in our region on the profile using an API)
- Ability to add an outline image to the profile photo
- Add an animation showing the person speaking (a circle around the pp, I found it disturbing at first and it seems to me to be a feasible addition)

## SERVER
- Add a Forum type room (a bit like on discord)
- Server Shop System
- Authorize Markdown on Channels & Categories
- Propose a Badge visible for grades **only on the server** (because badges are generally reserved for staff members)

## SETTINGS
- Customization of notifications (The user can import a sound with a defined size/duration)
- Manage the sound intensity in the *voice options* (currently it is only possible to speak)

I am open to any improvement suggestions and critical opinions.
and if you want to contact me for more details on Revolt
→ oxyblade#4246

**Comment by SixtyEmeralds** [comment]
Customizable sound notifications is something that completely fell to the wayside in Discord and I'd really like to see that addressed here.  I really want to be able to tell when someone critical is messaging me, and when someone is pinging me in a server, and I'd like to be able to define what that sound IS.
